### PR TITLE
Add 6 HHS non-.gov domains

### PR DESCRIPTION
**Pull request by climber-girl**

### CODE REVIEW

File: 2_govt_urls_federal_only.csv [hunk 1]
```diff
@@ -300,6 +302,9 @@ potomaccfc.org,Combined Federal Campaign,Federal Government,Virginia,accepted te
...
 project-open-data.github.io,,Federal Government,,accepted term,"Project Open Data, Executive Office of the President -- Note added 7/26/13 18:47",#N/A,7/26/13 18:47
 psctool.us,,Federal Government,,accepted term,"Product and Service Code (PSC) Selection Tool, a tool designed to help you navigate and select PSCs quickly and accurately. The tool allows users to find the right PSC using DOD's taxonomy, an intuitive framework that groups similar PSCs together. The taxonomy divides PSCs into sixteen portfolio groups: nine services groups and seven products groups. For more information on DOD's taxonomy, see the FY2012 http://www.acq.osd.mil/dpap/policy/policyvault/USA004219-12-DPAP.pdf memo announcing DOD's taxonomy and the full http://www.acq.osd.mil/dpap/ss/docs/fy11-dpap-portfolio_group_lookup-20120716.xlsx taxonomy-to-PSC map. -- Note added 8/20/14 10:36",https://psctool.us/,8/20/14 10:34
+psoppc.com,Health and Human Services (HHS),Federal Government,,accepted term,PSOPPC non-.govs have been verified as belonging to HHS by checking the about pages/certificates. PSOPPC was created by HHS' Agency for Healthcare Research and Quality (AHRQ).,,1/23/20 16:24
```
Comment (dawnpm) [inline 2_govt_urls_federal_only.csv]: Please add, for this and the following two entries, the homepage URL in the second to last column, e.g.,

psoppc.com,Health and Human Services (HHS),Federal Government,,accepted term,PSOPPC non-.govs have been verified as belonging to HHS by checking the about pages/certificates. PSOPPC was created by HHS' Agency for Healthcare Research and Quality (AHRQ).,https://www.psoppc.org/psoppc_web,1/23/20 16:24

File: 2_govt_urls_federal_only.csv [hunk 2]
```diff
@@ -386,4 +392,4 @@ westernmarylandcfc.org,Combined Federal Campaign,Federal Government,,accepted te
...
\ No newline at end of file
+yesprograms.org,,Federal Government,,accepted term,"YES Program - The Kennedy-Lugar Youth Exchange and Study (YES) program was established by Congress in October, 2002 in response to the events of September 11, 2001. The program is funded through the U.S. Department of State and sponsored by the Bureau of Educational & Cultural Affairs (ECA) to provide scholarships for high school students from countries with significant Muslim populations to spend up to one academic year in the U.S. Students live with host families, attend high school, engage in activities to learn about American society and values, acquire leadership skills, and help educate Americans about their countries and cultures. Starting in 2009, the YES Abroad program was established to provide a similar experience for U.S. students (15-18 years) to spend an academic year in select YES countries. -- Note added 7/13/15 14:22",http://www.yesprograms.org/,7/13/15 14:22
```
Comment (dawnpm) [inline 2_govt_urls_federal_only.csv]: Please move this entry to the quasi-governmental list. Please also add "Kennedy-Lugar Youth Exchange and Study" in column 2.

File: 2_govt_urls_federal_only.csv (hunk 1 above)
Comment (dawnpm) [reply 2_govt_urls_federal_only.csv:305]: @climber-girl, is there an active domain for these that you can add, as requested? Thanks, hope you're well.

Comment (climber-girl) [reply 2_govt_urls_federal_only.csv:305]: Sorry, I don't know... I don't own these domains, had just seen them in an email from HHS, so your team may want to reach out to them if you need that information.

**Review by Amberbblake — COMMENTED** [review]
ok wow

**Pull request closed without merging**